FluidNC/src/ProcessSettings.cpp

```diff
@@ -644,8 +644,10 @@ static Error xmodem_send(const char* value, WebUI::AuthenticationLevel auth_leve
         log_info("Cannot open " << value);
         return Error::DownloadFailed;
     }
+    bool prevAddCr = out.setCr(false);
```

Comment:
How about changing the name to oldCr so it is the same as the corresponding code in xmodem_receive() ?  With that change I will merge the change.  Thanks for debugging it.